test/unit/rules/no-aria-unsupported-elements-test.js

```diff
@@ -0,0 +1,72 @@
+import generateRuleTests from '../../helpers/rule-test-harness.js';
+
+generateRuleTests({
+  name: 'no-aria-unsupported-elements',
+
+  config: true,
+
+  good: ['<meta charset="UTF-8" />', '<html lang="en"></html>', '<script></script>'],
```

Comment:
Valid test cases we need to add to ensure other elements aren't restricted:
* `<div></div>`
* `<div aria-foo="true"></div>`
* `<div role="foo"></div>`

Reply:
Good shout out. Added these cases.